### PR TITLE
Blood tomatoes now produce O- blood.

### PR DESCRIPTION
**Pull request by harmonyn**
OOC: Bob Dobbington: While you're at it you should make the blood in blood tomatoes O- so they actually have a use besides pissing the janitor off

It is done. Never let it be said that coders do not listen to the players.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,4 @@
 *.lk
 *.int
 *.backup
-*.int
```
Comment (duncathan) [inline .gitignore]: you and your gitignore pushing :^)

Comment (bgobandit) [inline .gitignore]: I'M NOT COMMITTING IT I DON'T KNOW WHY THIS IS HAPPENING REEEEEEEEEEEE

Comment (Jalleo) [inline .gitignore]: Ask @oranges @mrstonedone .

**Comment by Incoming5643** [comment]
I will embrace my true Italian heritage and completely replace my blood with tomato paste

**Comment by MrPerson** [comment]
Mama mia!

File: .gitignore (hunk 1 above)
Comment (optimumtact) [inline .gitignore]: you have a commit on your branch that isn't on master that changes gitignore.

You should probably do a fresh checkout of upstream/master, and cherry-pick only the changes

Unfortunately you can't .gitignore the tgstation.int as it's already comitted, instead use this command

git update-index --assume-unchanged tgstation.int to ignore forever more all local changes to tgstation.int

**Comment by tuypo1** [comment]
:+1:

**Comment by Bawhoppen** [comment]
how is this a balance tag

**Comment by bgobandit** [comment]
the same tagging logic by which any edits to text are "grammar and formating"

**Comment by AnonymousNow** [comment]
Clever and desired.

**Pull request merged**